partner_second_lastname/README.rst

```diff
@@ -24,19 +26,40 @@ To use this module, you need to:
 If you directly enter the full name instead of entering the other fields
 separately (maybe from other form), this module will try to guess the best
 match for your input and split it between firstname, lastname and second
-lastname.
+lastname using an inverse function.
+
+You can configure some common name patterns for the inverse function
+in Settings > Configuration > General settings:
 
-If the name you enter is in the form *Firstname Lastname1 Lastname2*, it will
-be split as such. If you use a comma, it will understand it as *Lastname1
-Lastname2, Firstname*.
+* Lastname SecondLastname Firstname: For example 'Anderson Lavarge Robert'
+* Lastname SecondLastname, Firstname: For example 'Anderson Lavarge, Robert'
+* Firstname Lastname SecondLastname: For example 'Robert Anderson Lavarge'
+
+After applying the changes, you can recalculate all partners name clicking
+"Recalculate names" button. Note: This process could take so much time depending
+how many partners there are in database.
 
 If you can, always enter it manually please. Automatic guessing could fail for
 you easily in some corner cases.
 
+You can use *_get_inverse_name* method to get firstname, lastname and
+second lastname from a simple string and also *_get_computed_name* to get a
+name form the firstname, lastname and second lastname.
+These methods can be overridden to change the format specified above.
```

Comment:
Development section not exists in OCA template. I've moved it to Configuration section